docs/docs/reference/operator/contract.mdx

```diff
@@ -43,6 +43,7 @@ A full example of a Workflow Contract looks like
 | `envAllowList`                  | no       | List of environment variables that will be resolved and injected in the attestation                                                                                          |
 | [`runner`](#runner-context)     | no       | Specific runner type associated with this contract. If not set, this contract will be valid to be run `anywhere` but you'll miss out some of [its benefits](#runner-context) |
 | `annotations`                   | no       | Name/Value pairs of arbitrary annotations that will be added to the attestation. **If the value is not provided, it will be required during the attestation process.**       |
+| [`policies`](#policy-attachments)                   | no       | Attachments to existing Chainloop policies. See [policies reference guide](http://localhost:3000/reference/policies) for more information       |
```

Comment:
This link might be wrong

Reply:
It's a section at the bottom of the page.

Reply:
What I mean is that it's pointing to `localhost:3000` is that intended?

Reply:
ahhhh, sorry, you are right, I was looking at [`policies`](#policy-attachments)

Reply:
Fixed, thanks for catching it